node_modules/oae-admin/usermanagement/usermanagement.html

```diff
@@ -8,6 +8,7 @@
 
```

Comment:
Can you share the user interface prototypes for this, please? I just realised I haven't seen those yet :)